### DISCUSSION TITLE
How do I know which version of cypress-cucumber-preprocessor to use for a given Angular/Node/npm versions?

### DISCUSSION THREAD

**Discussion by dbnex14** [badeball/cypress-cucumber-preprocessor#1143] [Q&A]
I use following versions and am not ready to upgrade Angular at this time:

1. Angular CLI: 14.2.11
2. Node: 16.18.1
3. Package Manager: npm 8.19.2
4. OS: win32 x64

I use **cypress-cucumber-preprocessor 4.3.1** but I find that every call to cy.visit(url) like in

```
Given ('I navigate to Url {string}', (url) => {
    cy.visit(url)
})
```

, is extremely slow so I am trying to figure out why is this happening.

Based on this page, cypress-cucumber-preprocessor 4.3.1 should no longer be used.  

My question is how do I find which version of cypress-cucumber-preprocessor can I use with above versions?

**Comment by badeball** [comment]
The preprocessor doesn't depend on a particular version of angular, windows or npm, only _node_. I don't maintain a compatibility matrix with supported node versions, so my best advice is to just try.
